commands/blpop.md

```diff
@@ -57,7 +57,7 @@ What happens is that the command performing multiple pushes is executed, and *on
     Client A:   BLPOP foo 0
     Client B:   LPUSH foo a b c
 
-If the above condition happens using a Redis 2.6 server or greater, Client **A** will be served with the `c` element, because after the `LPUSH` command the list contains `c,b,a`, so taking an element from the left means to return `c`.
+If the above condition happens using a Redis OSS 2.6 server or greater, Client **A** will be served with the `c` element, because after the `LPUSH` command the list contains `c,b,a`, so taking an element from the left means to return `c`.
```

Comment:
"OSS Redis" or "Redis OSS"?
```suggestion
If the above condition happens using a OSS Redis 2.6 server or greater, Client **A** will be served with the `c` element, because after the `LPUSH` command the list contains `c,b,a`, so taking an element from the left means to return `c`.
```

Reply:
IANAL but I believe we must to use "Redis OSS" according to https://redis.io/legal/trademark-policy/, or maybe even "Redis® OSS".

> Offering an XYZ software, which is an altered, modified, or combined copy of Redis OSS, including Redis as a cloud service or as a container service fully complying with the Redis OSS API. In the product description, you may state that “XYZ software is compatible with the Redis® OSS API” or “Compatible with legacy Redis® OSS.” No other term or description of your software is allowed.

(...)

> Attribution. Any permitted use of the Trademarks, as indicated above, must comply with the following provisions:
>
> - You should always add the R mark (®) to “Redis”, and an asterisk (*) to the first mention on a website as part of a product description;
> - On each page where “Redis®*” is used – add the following legend in a noticeable and readable format: “*Redis is a registered trademark of Redis Ltd. Any rights therein are reserved to Redis Ltd. Any use by <company XYZ> is for referential purposes only and does not indicate any sponsorship, endorsement or affiliation between Redis and <company XYZ>”; and
> - Sections i. and ii. above apply to any appearance of the word “Redis” in: (a) any web page, gated or un-gated; (b) any marketing collateral, white paper, or other promotional material, whether printed or electronic; and (c) any advertisement.

I can't say I understand the extents of this, but possibly it applies only to a product descriptions and open source software are not products, but what do i know...

Reply:
We should remove the references to 2.x though, but I did this PR using automatic replacements, as mentioned. We can go through them and remove them later. If the trademark policy makes it hard, we can just remove all references, but let's find out first.

Reply:
I merged this now, but it's easy to replace all occurrences of "Redis OSS" whenever we know what we have to write.

Reply:
SGTM. 

>I can't say I understand the extents of this, but possibly it applies only to a product descriptions and open source software are not products, but what do i know...

I can go with "Redis OSS" too as long as we are consistent.

> We should remove the references to 2.x though, but I did this PR using automatic replacements, as mentioned.

I noticed that too but agreed with you that we should do one thing at a time.